Scripts/Items/Containers/LockableContainer.cs

```diff
@@ -300,6 +300,17 @@ public override bool TryDropItem(Mobile from, Item dropped, bool sendFullMessage
 
         public override bool OnDragDropInto(Mobile from, Item item, Point3D p)
         {
+            if (this is SecretChest sc)
```

Comment:
Can we override OnDragDropInto in SecretChest.cs?

Reply:
I derived the SecretChest class from LockableContainer. There is a problem because I use the Locked parameter in common.